candump.c

```diff
@@ -134,7 +134,7 @@ static void print_usage(char *prg)
 	fprintf(stderr, "         -a          (enable additional ASCII output)\n");
 	fprintf(stderr, "         -S          (swap byte order in printed CAN data[] - marked with '%c' )\n", SWAP_DELIMITER);
 	fprintf(stderr, "         -s <level>  (silent mode - %d: off (default) %d: animation %d: silent)\n", SILENT_OFF, SILENT_ANI, SILENT_ON);
-	fprintf(stderr, "         -l          (log CAN-frames into file. Sets '-s %d' by default)\n", SILENT_ON);
+	fprintf(stderr, "         -l <fname>  (log CAN-frames into file. Sets '-s %d' by default)\n", SILENT_ON);
```

Comment:
What happens to `candump -l vcan0 vcan1` ??
I don't see, that this way to add an 'optional' filename does not work as it breaks the current behaviour.

Reply:
Yeah I see, `candump -l vcan0 vcan1` would change to `candump -l -- vcan0 vcan1`. So breaking compatibility is no option? Another solution would be to introduce a new depending command line parameter, what would be kind of ugly, I guess... Any other ideas?

Reply:
Adding options is ok, changing options which might break peoples scripts not.
Even if it looks ugly I would suggest a different option, e.g. `-F <fname> (log into <fname>)` or `-f <fname>` ...

Reply:
> Another solution would be to introduce a new depending command line parameter, what would be kind of ugly, I guess... Any other ideas perhaps?

What about this suggestion? Would it fit for you?
https://github.com/hartkopp/can-utils/commit/74d9feb1044a0ccae63e93ac828b2fdecc012ae6

Reply:
Looks great.

Reply:
Thanks @xR3b0rn !
The new option is now applied in master branch.